src/App.js

```diff
@@ -1,9 +1,78 @@
 import "./App.css";
 
 const App = () => {
+  [
```

Comment:
Assign this to a const:

```suggestion
  const matches = [
```